### ISSUE TITLE
Template file does not get downloaded when installing

### ISSUE THREAD

**Issue by sergiodeveloper** [newrelic/serverless-newrelic-lambda-layers#75]
Error when trying to deploy Serverless application.

Error message:
```
Serverless: The required NewRelicLambdaIntegrationRole cannot be found; Creating Stack with NewRelicLambdaIntegrationRole.
Serverless: Something went wrong while creating NewRelicLambdaIntegrationRole: Error: ENOENT: no such file or directory, open '/home/username/projectname/node_modules/serverless-newrelic-lambda-layers/templates/nr-lambda-integration-role.yaml'
```

The template file `templates/nr-lambda-integration-role.yaml` does not get downloaded when installing dependency with `npm install`.

This bug affects all releases greater than `0.1.20`

The source of this bug may be [this line](https://github.com/newrelic/serverless-newrelic-lambda-layers/blob/ba5122f6ff589f66424a5221d18bc6437762b82c/src/utils.ts#L42), which tries to access the directory "templates", that doesn't exist in the installed module; the only directory under the installed module is "dist".

**Comment by kolanos** [comment]
@sergiodeveloper Thanks for reporting this, investigating now.

**Comment by kolanos** [comment]
@sergiodeveloper This should be fixed in `v0.2.5`. Thanks again.
